app/views/content/teacher-training-advisers.md

```diff
@@ -29,88 +16,115 @@ keywords:
     - preparation
 quote:
     q-tta-one:
-      text: "It's tailor-made to individual needs. I received an abundance of help."
-      inline: right
+      text: "They would answer any question I had, small or big, in a relaxed environment."
+      background: "grey"
     q-tta-two:
-      text: "[My adviser] was extremely helpful as well as thoughtful and understanding."
+      text: "They were a great sounding board for my concerns and queries."
+      background: "grey"
+    q-tta-three:
+      text: "Lisa really boosted my confidence – she gave me great advice and tips."
+      background: "grey"
+    q-tta-four:
+      text: "My adviser helped not only to remind me of the basics, but also to point out some of the things which are different from when I left 12 years ago."
+      background: "grey"
 calls_to_action:
   get-a-tta:
     name: simple
     arguments:
-      title: Get a teacher training adviser
-      text: "Teacher training advisers can help you with everything related to applying for teacher training."
+      title: Get an adviser
+      text: "Answer a few questions about your situation and we'll match you with an adviser."
       link_text: "Get a teacher training adviser"
       link_target: "/tta-service"
       icon: "icon-person"
-youtube_video:
-  about-teacher-training-advisers-video:
-    id: ZaGL8c4FkLA
-    title: A video about what Teacher Training Advisers do
-  about-teacher-training-advisers-experience-video:
-    id: T9Bhcaa6LJ4
-    title: A video about what teaching experience our Teacher Training Advisers have
 ---
-Teacher training advisers give you free, one-to-one help and support. 
+Teacher training advisers give you free, one-to-one support, whether you're just thinking about teaching or you're ready to apply.
 
-They're all former teachers with lots of experience, so they know the process inside out.
-
-## What an adviser can do for you
+They have years of teaching experience and can answer all your questions about life in the classroom, as well as how you can get there.
 
 $q-tta-one$
 
-An adviser can give you as much or as little help as you need. 
+## How does it work?
 
-Other trainee teachers say they are particularly helpful with: 
+You’ll be matched with your own dedicated adviser. You’ll have the same adviser throughout your journey into teaching.
 
-* arranging school experience
-* seeking out financial support
-* writing a personal statement
-* preparing for interviews
+How often you talk to them is up to you – this could be every week for more than a year, or just a few times for some one-to-one support.
+
+You can chat by phone, or by:
+
+* email
+* text
+* WhatsApp
+
+## How your adviser can help you
```

Comment:
This is the only h2 on the page that isn't a question - should it be reworded to a question for consistency @EllieNodder?